diff --git a/node_modules/app-builder-lib/out/codeSign/windowsCodeSign.js b/node_modules/app-builder-lib/out/codeSign/windowsCodeSign.js
index a379ba4..b8d2c24 100644
--- a/node_modules/app-builder-lib/out/codeSign/windowsCodeSign.js
+++ b/node_modules/app-builder-lib/out/codeSign/windowsCodeSign.js
@@ -356,7 +356,7 @@ function getWinSignTool(vendorPath) {
   if (isOldWin6()) {
     return path.join(vendorPath, "windows-6", "signtool.exe");
   } else {
-    return path.join(vendorPath, "windows-10", process.arch, "signtool.exe");
+    return path.join(vendorPath, "windows-10", "x64", "signtool.exe");
   }
 }
 
diff --git a/node_modules/app-builder-lib/out/util/yarn.js b/node_modules/app-builder-lib/out/util/yarn.js
index 0182f9e..fb3f6d6 100644
--- a/node_modules/app-builder-lib/out/util/yarn.js
+++ b/node_modules/app-builder-lib/out/util/yarn.js
@@ -125,7 +125,7 @@ function installDependencies(appDir, options) {
     appDir
   }, `installing production dependencies`);
 
-  let execPath = process.env.npm_execpath || process.env.NPM_CLI_JS;
+  let execPath = require('child_process').spawnSync('which', ['npm']).stdout.toString().trim();
   const execArgs = ["install"];
   const npmUserAgent = process.env["npm_config_user_agent"];
   const isYarn2 = npmUserAgent != null && npmUserAgent.startsWith("yarn/2.");
@@ -194,7 +194,7 @@ async function rebuild(appDir, options) {
     platform: options.platform || process.platform,
     arch: options.arch || process.arch,
     additionalArgs: options.additionalArgs,
-    execPath: process.env.npm_execpath || process.env.NPM_CLI_JS,
+    execPath: require('child_process').spawnSync('which', ['npm']).stdout.toString().trim(),
     buildFromSource: options.buildFromSource === true
   };
   const env = getGypEnv(options.frameworkInfo, configuration.platform, configuration.arch, options.buildFromSource === true);
